umm-spec-lib/resources/json-schemas/variable/umm/v1.9/umm-var-json-schema.json

```diff
@@ -0,0 +1,604 @@
+      "$ref": "#/definitions/MetadataSpecificationType"
+    }
+  },
+  "required": [
+    "Name",
+    "LongName",
+    "Definition",
+    "MetadataSpecification"
+  ],
+  "definitions": {
+    "AdditionalIdentifierType": {
+      "type": "object",
+      "additionalProperties": false,
+      "description": "The elements of this section apply to an additional identifier.",
+      "properties": {
+        "Identifier": {
+          "description": "The actual identifier.",
+          "type": "string",
+          "minLength": 1,
+          "maxLength": 1024
+        },
+        "Description": {
+          "description": "This element describes to a person or machine what the identifier is called. e.g., if the identifier is 1057.2345/asfb then the Description should be DOI or Digital Object Identifier.",
+          "type": "string",
+          "minLength": 0,
+          "maxLength": 4000
+        }
+      },
+      "required": ["Identifier"]
+    },
+    "IndexRangesType": {
+      "type": "object",
+      "additionalProperties": false,
+      "description": "The index ranges consist of a LatRange and a LonRange.",
+      "properties": {
+        "LatRange": {
+          "description": "The LatRange consists of an index range for latitude.",
+          "type": "array",
+          "items": {
+            "type": "number"
+          },
+          "minItems": 2,
+          "maxItems": 2
+        },
+        "LonRange": {
+          "description": "The LonRange consists of an index range for longitude.",
+          "type": "array",
+          "items": {
+            "type": "number"
+          },
+          "minItems": 2,
+          "maxItems": 2
+        }
+      },
+      "required": ["LatRange", "LonRange"]
+    },
+    "MeasurementIdentifierType": {
+      "type": "object",
+      "additionalProperties": false,
+      "description": "The elements of this section allow authors to provide community sourced words or phrases to further describe the variable data.",
+      "properties": {
+        "MeasurementContextMedium": {
+          "description": "This element describes the context/medium within which the measurement was made.",
+          "type": "string",
+          "minLength": 1,
+          "maxLength": 1024
+        },
+        "MeasurementContextMediumURI": {
+          "description": "This element contains the URI for the context/medium.",
+          "type": "string",
+          "minLength": 1,
+          "maxLength": 4000
+        },
+        "MeasurementObject": {
+          "description": "This element describes the object which was measured.",
+          "type": "string",
+          "minLength": 1,
+          "maxLength": 1024
+        },
+        "MeasurementObjectURI": {
+          "description": "This element contains the URI for the object which was measured.",
```

Comment:
URI can be enforced

Reply:
yes, we could change the type to be a URI, but first I would want to check all of the data, since this field has been around for 4 years.

Reply:
After fixing 5 records that existed in ES and not oracle, and after getting the data and analyzing it, I saw that I could put the uri constraint on this field, so I did.